tests/components/FilterSearch.test.tsx

```diff
@@ -0,0 +1,256 @@
+import { FilterSearch } from '../../src/components/FilterSearch';
+import { render, screen } from '@testing-library/react';
+import * as useSynchronizedRequestFunctions from '../../src/hooks/useSynchronizedRequest';
+import userEvent from '@testing-library/user-event';
+import * as searchOperations from '../../src/utils/search-operations';
+import { mockAnswersActions } from '../__utils__/mocks';
+import { mockedFilterSearchResponse, mockedFilterSearchResponseNoResults, mockedFilterSearchResponseNoLabels} from '../../tests/__fixtures__/data/filtersearch';
+
+jest.mock('@yext/answers-headless-react');
+
+const mockedExecuteFilterSearch = jest.fn();
+const setFilterOption = jest.fn();
+const setOffset = jest.fn();
+const searchFieldsProp = {
```

Comment:
nit: let's make this an array so you don't have to put brackets around in `searchFields={**[**searchFieldsProp**]**}` each time.

Reply:
fixed!